docs/how-to/dummy-data.md

```diff
@@ -46,7 +46,7 @@ You can provide a dummy dataset file in the following formats.
 :warning: Your dummy dataset file must have the relevant file extension shown in the table
 above.
 
-For example, take this dataset definition from the tutorial:
+For example, take this dataset definition from the homepage:
```

Comment:
In the context of the main opensafely docs (where users will be), I don't think I'd know what the "homepage" meant. I'd just change to "For example, take this dataset definition:", since we're reproducing it again anyway.